content/docs-v1/citation-bibliographies.md

```diff
@@ -33,72 +31,102 @@ entries:
     full: "de Goncourt, Edmond. *Journal des Goncourt: Mémoires de la vie littéraire.* Paris; G. Charpentier et cie, 1851."
 ```
 
-### Sort Order
+{{< q-class "box tip" >}}
+- By default, Quire will sort bibliographic entries alphabetically by the text of the `full` reference. In the event that you want to override alphabetical sorting, you can add a `sort_as` attribute to the YAML entry to change the ordering.
+{{< /q-class >}}
 
-By default, Quire will sort bibliographic entries alphabetically by the text of the `full` reference. However, you may find in some cases that Quire’s default sort method has its drawbacks. For example, the presence of diacritics, lowercase names, or two author names in a citation may affect the proper ordering of your references. Use the `sort` value to override any ordering issues created by the `full` value:
-
-```yaml
-entries:
-  - id: "Mütter 1851"
-    full: "Mütter, Edmond. *Journal des Goncourt: Mémoires de la vie littéraire.* Paris; G. Charpentier et cie, 1851."
-    sort: "Mutter"
-```
-
-These references can then be called individually from within text using the `q-cite` shortcode, or in their entirety as a generated bibliography using the `q-bibliography` shortcode. Both of which are detailed below.
+These references can then be called individually from within text using the `{% cite %}` shortcode or they can appear in their entirety as a page of your publication by adding a `bibliography.md` file to your project. Both options are detailed below.
```

Comment:
> by adding a `bibliography.md` file to your project

The file doesn't need to be named `bibliography.md` it just needs to have `layout: bibliography`